python/googleapis/python-multi/Dockerfile

```diff
@@ -129,4 +129,22 @@ ENV PATH /google-cloud-sdk/bin:$PATH
 # Enable redis-server on boot.
 RUN sudo systemctl enable redis-server.service
 
+# Install docker
+RUN apt-get update \
+  && apt-get install -y \
+     apt-transport-https \
+     ca-certificates \
+     curl \
+     software-properties-common \
+     sudo
```

Comment:
These should probably be added to the install set at L28 (a few are already there).